database/enums.py

```diff
@@ -0,0 +1,21 @@
+from enum import Enum, unique
+@unique
+class Role(Enum):
+    SHIFTER = 'Shifter'
+    COORDINATOR = 'Coordinator'
+    ADMIN = 'Admin'
```

Comment:
Standardize formatting
```suggestion
    SHIFTER = "Shifter"
    COORDINATOR = "Coordinator"
    ADMIN = "Admin"
```